website/docs/d/tg_route_report.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+
+subcategory: "Transit Gateway"
+layout: "ibm"
+page_title: "IBM : tg_route_report"
+description: |-
+  Manages IBM Cloud Infrastructure Transit Gateway Route Report.
+---
+
+# ibm_tg_route_report
+Retrieve information of an existing IBM Cloud infrastructure transit gateway route report as a read only data source. For more information about Transit Gateway Route Reports, see [generating and viewing a route report](https://cloud.ibm.com/docs/transit-gateway?topic=transit-gateway-route-reports&interface=ui#generate-route-report-ui).
+
+## Example usage
+
+```terraform
+data "ibm_tg_route_report" "tg_route_report" {
+    gateway = ibm_tg_gateway.new_tg_gw.
+    id = ibm_tg_route_report_test_tg_route_report.route_report_id
```

Comment:
update the documentation with latest changes its not "id"

Reply:
any update on this its not fixed..

Reply:
This has been fixed. Can you please review again @hkantare?